<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QVTKOpenGLNativeWidget" name="qvtkWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="filters_btns">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_filters">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>Filters</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_sor">
        <property name="text">
         <string>SOR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ror">
        <property name="text">
         <string>ROR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_median">
        <property name="text">
         <string>Median</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_voxel_grid">
        <property name="text">
         <string>VoxelGrid</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_mls">
        <property name="text">
         <string>MLS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_mor">
        <property name="text">
         <string>MOR</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_filter_with_normals">
        <property name="text">
         <string>Filters with normals</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_ssn">
        <property name="text">
         <string>SSN</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_shadow_points">
        <property name="text">
         <string>Shadow points</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>10</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btn_reset_points">
        <property name="text">
         <string>Reset to original</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadPCD"/>
    <addaction name="actionSaveCurrPointCloud"/>
   </widget>
   <widget class="QMenu" name="menuCamera">
    <property name="title">
     <string>Camera</string>
    </property>
    <addaction name="actionCameraReset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCamera"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoadPCD">
   <property name="text">
    <string>Load PCD</string>
   </property>
  </action>
  <action name="actionSaveCurrPointCloud">
   <property name="text">
    <string>Save current point cloud</string>
   </property>
  </action>
  <action name="actionCameraReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
